### PR TITLE
initial commit project 2 part 1 complete

### PR DESCRIPTION
**Pull request by jackdgds**

### CODE REVIEW

File: main.cpp [hunk 1]
```diff
@@ -59,10 +60,29 @@ void variableDeclarations()
...
+    unsigned int cost = 75;
+    unsigned int score = 70;
+    
+    
+    
     
-
-    
-    ignoreUnused(number); //passing each variable declared to the ignoreUnused() function
+    ignoreUnused(5, 20, 100, 2.5f, 1.5f, 195.7f, true, false, true, 5.999999, 5.0, 5, 'A', 'E', 'B', 150, 75, 70); //passing each variable declared to the ignoreUnused() function
```
Comment (matkatmusic) [inline main.cpp]: Pass the VARIABLES that you declared, not the values.
- [x] revise

File: main.cpp [hunk 2]
```diff
@@ -77,45 +97,85 @@ bool rentACar(int rentalDuration, int carType = 0)  //function declaration with
...
 
 /*
- 1)
+ 1) 
  */
-
+bool rentHotelRoom(int rentalLength, int suiteType = 2)
+{
+    ignoreUnused(rentalLength,suiteType);
+    return { };
```
Comment (matkatmusic) [inline main.cpp]: add space after `,` to make the code more readable
- [x] revise

File: main.cpp [hunk 3]
```diff
@@ -77,45 +97,85 @@ bool rentACar(int rentalDuration, int carType = 0)  //function declaration with
...
+bool isInTune(char tuning = 'A')
+{
+    ignoreUnused(tuning);
+    return { };
+}
 /*
  4)
  */
-
+int battingAverage(float leadOff = 0.290f, float cleanUp = 0.305f)
```
Comment (matkatmusic) [inline main.cpp]: choose a function name that contains a verb.
`battingAverage` is a property that can be represented with a primitive type. 
- [x] revise

File: main.cpp [hunk 4]
```diff
@@ -77,45 +97,85 @@ bool rentACar(int rentalDuration, int carType = 0)  //function declaration with
...
+{
+    ignoreUnused(leadOff, cleanUp);
+    return { };
+}
 /*
  5)
  */
-
+int savePercentage(float backupGoalie = 0.920f, float callupGoalie = 0.850f, float franchiseGoalie = 0.975f)
+{
```
Comment (matkatmusic) [inline main.cpp]: choose a function name that contains a verb.
- [x] revise

File: main.cpp [hunk 5]
```diff
@@ -77,45 +97,85 @@ bool rentACar(int rentalDuration, int carType = 0)  //function declaration with
...
+    ignoreUnused(backupGoalie, callupGoalie,     
+    franchiseGoalie);
+    return { };
+}
 /*
  6)
  */
-
+int earnedRunAverage(float starter = 3.45f, float reliever = 4.69f, float closer = 1.67f)
+{
```
Comment (matkatmusic) [inline main.cpp]: choose a function name that contains a verb.
- [x] revise

File: main.cpp [hunk 6]
```diff
@@ -77,45 +97,85 @@ bool rentACar(int rentalDuration, int carType = 0)  //function declaration with
...
+    ignoreUnused(starter, reliever, closer);
+    return { };
+}
 /*
  7)
  */
-
+void nhlTeamsGeographically(int usa = 25, int canada = 7)
+{
```
Comment (matkatmusic) [inline main.cpp]: choose a function name that contains a verb.
- [x] revise

File: main.cpp [hunk 7]
```diff
@@ -77,45 +97,85 @@ bool rentACar(int rentalDuration, int carType = 0)  //function declaration with
...
+    ignoreUnused(usa, canada);
+}
 /*
  8)
  */
-
+bool gameOn(unsigned int homeTeam = 3, unsigned int awayTeam = 0)
+{
```
Comment (matkatmusic) [inline main.cpp]: choose a function name that contains a verb.
- [x] revise

File: main.cpp [hunk 8]
```diff
@@ -77,45 +97,85 @@ bool rentACar(int rentalDuration, int carType = 0)  //function declaration with
...
+    ignoreUnused(homeTeam, awayTeam);
+    return { };
+}
 /*
  9)
  */
-
+bool gameCancelled(bool covidRules = true, char attendance = 'n' )
+{
```
Comment (matkatmusic) [inline main.cpp]: choose a function name that contains a verb.
- [x] revise

**Review by matkatmusic — CHANGES_REQUESTED** [review]

File: main.cpp (hunk 3 above)
Comment (jackdgds) [reply main.cpp]: Not sure if `battingAverage is a property that can be represented with a primitive type.` means you are asking me to change something besides adding a verb to the function name.

**Review by matkatmusic — APPROVED** [review]

**Pull request merged**